mev_inspect/arbitrages.py

```diff
@@ -52,6 +52,11 @@ def _get_arbitrages_from_swaps(swaps: List[Swap]) -> List[Arbitrage]:
         ]
         routes = _get_all_routes(start, end, potential_intermediate_swaps)
 
+        error = ""
+        for swap in swaps:
```

Comment:
Not all swaps will be involved in all arbs

Needs to be considered separately for each route I think

Like
```
for route in routes:
    error = None

    for swap in route:
        if swap.error is not None
             error = swap.error
 
     ...
     arb = Arbitrage(..., error=error)
```